birdhouse/components/stac-browser/default.env

```diff
@@ -27,6 +36,7 @@ OPTIONAL_VARS="
   \$STAC_BROWSER_DOCKER
   \$STAC_BROWSER_IMAGE
   \$STAC_BROWSER_IMAGE_URI
+  \$STAC_BROWSER_PATH_PREFIX
```

Comment:
Why does `STAC_BROWSER_PATH_PREFIX` need to be added to `DELAYED_EVAL`?  It does not make use of other variables.